### PR TITLE
Register page Added

### PR DESCRIPTION
**Pull request by AnandTringapps**
This pull request adds a basic HTML registration form with client-side validation using JavaScript. The form includes fields for username, email, password, and confirm password, with the following features:

Validation: Client-side validation ensures that all fields are filled out and that the passwords match before submission.
Styling: Basic styling is applied to the form for improved appearance and usability.
Feel free to review the code and provide any feedback or suggestions. Thank you!

### CODE REVIEW

File: Register.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    <input type="password" id="password" name="password">
+    <label for="confirmPassword">Confirm Password:</label>
+    <input type="password" id="confirmPassword" name="confirmPassword">
+    <input type="submit" value="Register">
+    <p id="error-msg" class="error"></p>
+</form>
+
+<script>
+function validateForm() {
+    var username = document.getElementById("username").value;
```
Comment (dinesh-ta) [inline Register.html]: Have to use const instead of var in the declaration.

Comment (AnandTringapps) [reply Register.html]: Sure, I will change this.

Comment (AnandTringapps) [reply Register.html]: Check it once pls.

**Review by dinesh-ta — APPROVED** [review]
Changes updated.

**Review by dinesh-ta — APPROVED** [review]
Changes updated.

**Review by dinesh-ta — APPROVED** [review]
Changes Updated

**Review by dinesh-ta — APPROVED** [review]

**Pull request merged**